app/src/main/res/values/dimens.xml

```diff
@@ -16,10 +16,17 @@
     <dimen name="padding_large">16dp</dimen>
     <dimen name="padding_extra_large">32dp</dimen>
     <dimen name="padding_extra_larger">64dp</dimen>
+    <dimen name="padding_mem_graph">54dp</dimen>
+    <dimen name="margin_large">60dp</dimen>
+    <dimen name="margin_extra_large">248dp</dimen>
```

Comment:
I guess we now need to follow some conventions regarding the sizes and colors. Else our resources file will simply bloat up. Instead of adding every dimension to our list we could have some predefined values which can be used everywhere because I see there are many values that are repeated throughout the file. Anyways I guess we would do it in an iteration sometime later. Let us get our app functioning now.

Reply:
Yeah exactly had similar thoughts. Currently was desperately working to make the UI better looking and functional. I agree we need to take care of these at a later stage.